js/main.js

```diff
@@ -202,38 +203,105 @@ var $home = document.querySelector('#home');
 
 $retry.addEventListener('click', resetQuiz)
 function resetQuiz(){
-    //reset data to default
-    data.currentNumber = 1;
-    data.correctPokemon = [];
-    data.wrongPokemon = [];
-    var $dots = document.querySelectorAll('.col-tenth')
-    for (var i = 0; i < $dots.length; i++){
-      $dots[i].innerHTML = '<i class="fas fa-circle"></i>'
-    }
-    var $quizContainer = document.querySelector('.quizContainer')
-    $quizContainer.remove();
-    $navH1.textContent = "Question " + data.currentNumber;
-    $quizModal.className = "modal-background hidden"
-    createQuizContainer();
-    generateFourRandomPokemonNumbers();
-    getPokemonPicture()
+  //reset data to default
+  data.currentNumber = 1;
+  data.correctPokemon = [];
+  data.wrongPokemon = [];
+  var $dots = document.querySelectorAll('.col-tenth')
+  for (var i = 0; i < $dots.length; i++){
+    $dots[i].innerHTML = '<i class="fas fa-circle"></i>'
+  }
+  var $quizContainer = document.querySelector('.quizContainer')
+  $quizContainer.remove();
+  $navH1.textContent = "Question " + data.currentNumber;
+  $quizModal.className = "modal-background hidden"
+  createQuizContainer();
+  generateFourRandomPokemonNumbers();
+  getPokemonPicture()
 }
 
 $home.addEventListener('click', clearQuiz);
 function clearQuiz(){
-    data.currentNumber = 1;
-    data.correctPokemon = [];
-    data.wrongPokemon = [];
-    data.trainerName = null;
-    var $dots = document.querySelectorAll('.col-tenth')
-    for (var i = 0; i < $dots.length; i++) {
-      $dots[i].innerHTML = '<i class="fas fa-circle"></i>'
+  data.currentNumber = 1;
+  data.correctPokemon = [];
+  data.wrongPokemon = [];
+  data.trainerName = null;
+  var $dots = document.querySelectorAll('.col-tenth')
+  for (var i = 0; i < $dots.length; i++) {
+    $dots[i].innerHTML = '<i class="fas fa-circle"></i>'
+  }
+  var $quizContainer = document.querySelector('.quizContainer')
+  $quizContainer.remove();
+  $homePage.className = "container"
+  $quizPage.className = "container hidden"
+  $quizModal.className = "modal-background hidden"
+  $body.className = ""
+  $navH1.textContent = "Pokemon Quiz Game";
+}
+
+
+var $playHistoryButton = document.querySelector('#playHistory');
+$playHistoryButton.addEventListener('click', switchToHistory)
+var $pastGamesView = document.querySelector('#pastGames');
+function switchToHistory(){
+  $homePage.className = "container hidden"
+  $pastGamesView.className = "container"
+  $body.className = "blueBackground"
+  $navH2.className = "navh2 pokemon-font"
+  createTrainerEntry();
+}
+$navH2.addEventListener('click', homeScreen)
+function homeScreen(){
+  $homePage.className = "container";
+  $pastGamesView.className = "container hidden";
+  $navH2.className = "navh2 pokemon-font hidden"
+  $body.className = ""
+  var $trainerData = document.querySelectorAll('.trainerData');
+  for (var i = 0; i < $trainerData.length; i++){
+    $trainerData[i].remove();
+  }
+}
+
+function createTrainerEntry(){
```

Comment:
You should make this function just generate a DOM tree for one game's history, then make a separate function to loop over the past games and generate a DOM tree for each one with this function, then `append` the `return` of that function to the DOM